client/sites-dashboard/components/wpcom-smp-commands.tsx

```diff
@@ -175,22 +194,22 @@ export const useCommandsArrayWpcom = ( {
 				},
 				filter: ( site: SiteExcerptData ) => site?.is_wpcom_atomic,
 			},
-			icon: <MaterialIcon icon="key" />,
+			icon: keyIcon,
 		},
 		{
-			name: 'openSshDetails',
-			label: __( 'Open SSH details' ),
-			searchLabel: __( 'open SSH details' ),
-			context: 'Opening SSH details',
-			callback: setStateCallback( 'openSshDetails' ),
+			name: 'openSshCredentials',
+			label: __( 'Open SFTP/SSH credentials' ),
+			searchLabel: __( 'open SFTP/SSH credentials' ),
+			context: 'Opening SFTP/SSH credentials',
+			callback: setStateCallback( 'openSshCredentials' ),
 			siteFunctions: {
 				onClick: ( { site, close }: { site: SiteExcerptData; close: () => void } ) => {
 					close();
 					navigate( `/hosting-config/${ site.slug }` );
```

Comment:
On Raycast I use: `#sftp-credentials`. Do you want to change it for consistency? It doesn't add much, just adds a little bit of scroll.